llvm/lib/RemoteCachingService/CMakeLists.txt

```diff
@@ -1,27 +1,26 @@
 if (LLVM_CAS_ENABLE_REMOTE_CACHE)
 add_subdirectory(RemoteCacheProto)
-add_subdirectory(RemoteCacheServer)
-add_subdirectory(Client)
-add_subdirectory(CAS)
 
 set(LLVM_LINK_COMPONENTS
   RemoteCacheKVProto
   RemoteCacheCASProto
-  RemoteCacheClient
-  GRPCRelayCAS
-  )
-else ()
-add_subdirectory(NullService)
+)
 
-set(LLVM_LINK_COMPONENTS
-  RemoteNullService
-  )
+  include_directories($<TARGET_PROPERTY:LLVMRemoteCacheKVProto,BINARY_DIR>)
+  include_directories($<TARGET_PROPERTY:LLVMRemoteCacheCASProto,BINARY_DIR>)
+
+  set(REMOTE_CACHE_SOURCE "Client/Client.cpp" "CAS/GRPCRelayCAS.cpp" "RemoteCacheServer/LLVMCASCacheProvider.cpp" "RemoteCacheServer/RemoteCacheServer.cpp")
```

Comment:
Can you match the indentation like other `set`?

Also not sure if quotation mark is needed.

Reply:
I thought clang-format would do it, but now that @akyrtzi wanted RemoteCacheServer to be it's own dylib, the set is only 66 characters long

Reply:
clang-format doesn't do CMake files. Usually it is formatted into one word on each line (see `set` above)
```
set(REMOTE_CACHE_SOURCE
  Client/Client.cpp
  CAS/GRPCReplayCAS.cpp
  ...
)
```
Something like this.

Reply:
And the indentation should be the same for all the lines in `if()`

Reply:
Okay sounds good, I can fix that!